pkg/server/api_test.go

```diff
@@ -32,21 +32,21 @@ type scenario struct {
 
 const expectedContentLength int = 32
 
-func setUserAgentOnReq(req *http.Request) *http.Request {
-	req.Header.Set("User-Agent", "Ignition/0.35.0")
+func setAcceptHeaderOnReq(req *http.Request) *http.Request {
```

Comment:
not knowing much here do these unit tests adequately test the helper funcs you modified above?

Reply:
the header is set on all (most) requests in the tests below, confirming it works. I've added a separate tests for the two header funcs for even more confidence.